src/components/usa-step-indicator/usa-step-indicator.stories.js

```diff
@@ -0,0 +1,66 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-step-indicator.twig";
+import {
+  Data,
+  CenterCountersSmData,
+  CenterCountersData,
+  CenterNoLabelData,
+  CenterData,
+  CountersSmData,
+  CountersData,
+  NoLabelsData,
+  ShortData,
+} from "./content";
+
+export default {
+  title: "Components/Step Indicator",
+  argTypes: {
+    modifier: {
+      control: "text",
+    },
+    currentStep: {
+      control: "number",
+    },
+    steps: {
+      // step: {
+      //   control: "text",
+      // },
+      // status: {
+      //   options: ["complete", "current", "incomplete"],
+      //   control: { type: "radio" },
+      // },
+      // label: {
+      //   control: "text",
+      // },
+    },
+  },
+};
```

Comment:
Proposal: set currentStep to a range type control so we can set a min and max

Reply:
Leaving this for the future / @mejiaj review